docs/usage.rst

```diff
@@ -2,11 +2,114 @@
+
+``root-dir``: Das Installationsverzeichnis von kitovu.
+
+Abschnitt ``connections``
+*************************
+
+``connections``: Die Verbindungen zu den Plattformen, von denen du deine Unterrichtsmaterialien synchronisierst, in der Regel ist das ein SMB-Skripteserver oder Moodle.
+
+``name``: Ein von dir frei wählbarer Name für die jeweilige Plattform.
+
+``plugin``: Der Name der Plattform, der kitovu intern verwendet, damit eine Verbindung zustande kommt. Derzeit gibt es die fixe Bezeichnungen ``smb`` oder ``moodle``.
+
+``username``: Dein Login-Name, womit du dich auch andernorts an der Schule einloggst, bestehend aus Vor- und Nachname. Moodle verlangt keinen Usernamen, dafür nach einem Sicherheitsschlüssel (s. weiter unten).
+
+Abschnitt ``subjects``
+**********************
+
+``subjects`` bezeichnet ein Unterrichtsmodul, das du an der Schule belegst, beispielsweise "Betriebssysteme 1" oder "Objektorientierte Programmierung in Java".
+
+``name``: Ein von dir frei wählbarer Name für das jeweilige Unterrichtsmodul.
+
+``sources``: Die Plattformen, von denen du die Unterrichtsmaterialien synchronisierst. Gewisse Unterrichtsmodule sind sowohl auf Moodle als auch auf dem Skripteserver zu finden - hier kannst du für jede Plattform einen separaten Eintrag erstellen.
+
+``connection``: Der Name der Plattform, den du weiter oben unter ``connections`` bzw. dort als ``name`` festgelegt hast.
+
+``remote-dir``: Das Verzeichnis, das synchronisiert werden soll. In Moodle entspricht dies dem Titel, den der Kurs im Browser trägt. Im folgenden Beispiel ist das "Betriebssysteme 2 FS2016":
+
+.. image:: images/moodle_names.png
+
+Synchronisation
+---------------
+
+Bei der ersten Synchronsiation fragt dich kitvou nach allen benötigten Passwörtern. Für den Skripteserver ist dies das Passwort, das du für alle anderen Dienste an der Schule verwendest.
+
+Moodle hingegen ist ein Spezialfall. Um dich erfolgreich mit Moodle verbinden zu können, fragt dich kitovu bei der Erstverwendung nach dem "Sicherheitsschlüssel". Den findest du folgendermassen:
+
+    #. Logge dich via Browser auf Moodle ein.
+    #. Navigiere zu den Einstellungen: klicke auf das Profilfoto oben rechts und wähle ``Einstellungen``, dann ``Sicherheitsschlüssel``.
+    #. Kopiere den Eintrag unter "Schlüssel", der mit dem Servie "Moodle mobile web service" beschrieben wird. Das ist eine lange Kette von Zeichen und Zahlen. **Gib auf keinen Fall diesen Sicherheitsschlüssel weiter und behalte ihn für dich!**
+    #. Füge den Sicherheitsschlüssel in kitovu ein, wenn dich das Programm danach fragt.
+
+Der FileCache
+-------------
+
+Wenn du Dateien synchronisierst, hält kitovu das in einem Log fest. Nach Ende eines Semesters bzw. nach Prüfungsende kannst du diese Datei wieder löschen - also wenn keines der Unterrichtsmodule des vergangenen Semesters mehr synchronisiert werden sollte. Du siehst, wo diese Datei gespeichert ist, indem du ``kitovu fileinfo`` auf der Kommandozeile eingibst.
```

Comment:
"Log" klingt hier IMHO immer noch seltsam (und wird von nicht-Informatikstudenten wohl kaum verstanden in dem Kontext).